packages/promotion/src/services/promotion-module.ts

```diff
@@ -19,8 +20,14 @@ import {
   PromotionService,
 } from "@services"
 import { joinerConfig } from "../joiner-config"
-import { CreateApplicationMethodDTO, CreatePromotionDTO } from "../types"
 import {
+  CreateApplicationMethodDTO,
+  CreatePromotionDTO,
+  UpdateApplicationMethodDTO,
+  UpdatePromotionDTO,
+} from "../types"
```

Comment:
question: Would it make sense for these types to live in `@medusajs/types` since they are public-facing?

Reply:
These are types for the internal services, not the public facing interfaces. eg. data gathered in the `CreatePromotionDTO` array would be sent to the `promotionService`. There is however a type called `CreatePromotionDTO` in `@medusjs/types` that is public facing.

Reply:
Ah, got it. I thought since this was the module service, we would use the types from `@medusajs/types`.